packages/react-aria-components/docs/Popover.mdx

```diff
@@ -42,16 +42,26 @@ type: component
 ## Example
 
 ```tsx example
-import {DialogTrigger, Popover, Dialog, Button, OverlayArrow} from 'react-aria-components';
+import {DialogTrigger, Popover, Dialog, Button, OverlayArrow, Heading, Switch} from 'react-aria-components';
 
 <DialogTrigger>
-  <Button>Open popover</Button>
+  <Button>Settings</Button>
   <Popover>
     <OverlayArrow>
       <svg width={12} height={12} viewBox="0 0 12 12"><path d="M0 0 L6 6 L12 0" /></svg>
     </OverlayArrow>
     <Dialog>
-      This is a popover.
+      <div className="flex-col">
```

Comment:
this example announces really oddly in Chrome with VO
<img width="688" alt="Screenshot 2023-10-30 at 12 26 05 PM" src="https://github.com/adobe/react-spectrum/assets/698229/68442efc-b984-423e-aebf-7b3f95b939b5">